### PR TITLE
vet-clinic-database

### PR DESCRIPTION
**Pull request by demix007**
## In this milestone, I implemented the following:

- Created a database named vet_clinic.
- Created a table named "animals" with the following columns and data types in the database:
    💠 id: integer
    💠 name: string
    💠 date_of_birth: date
    💠 escape_attempts: integer
    💠 neutered: boolean
    💠 weight_kg: decimal

- Copied the SQL used in creating the table into the schema.sql file.
- Inserted all the required data into the table columns, and added the SQL used for insertion into the data.sql file.
-  Queried the data in the table columns and added the SQL used for the query into the queries.sql file.
- The following are the required queries and screenshots from the SQL Shell terminal to display the query results.

1. Find all animals whose name ends in "mon".
2. List the name of all animals born between 2016 and 2019.
3. List the name of all neutered animals that have less than 3 escape attempts.
4. List the date of birth of all animals named either "Agumon".
5. List the name and escape attempts of animals that weigh more than 10.5kg.
6. Find all animals that are neutered.
7. Find all animals not named Gabumon.
8. Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg).

data_query(1)

data_query(2)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-01-01';
```
Comment (ShadyShawkat) [inline queries.sql]: - [x] Please make sure to select all animals that have a birth date between the years `2016` and `2019`, meaning that if they had a birth date in the year 2019 (until `2019/31/12`) they should be selected.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @demix007, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (demix007) [reply queries.sql]: Hello @ShadyShawkat 👋🏼 , 

Thank you for taking out time to review and recommend changes to this PR. The recommended changes will be implemented immediately. 👍🏼

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '01/01/2016' AND '01/01/2019';
```
Comment (t-yanick) [inline queries.sql]: - [ ] The last reviewer asked that your query should consider animals born in 2019, right up 12/31/2019. Your query statement should read like 
`SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';` 
or 
`SELECT name FROM animals WHERE date_of_birth BETWEEN '01/01/2016' AND '12/31/2019';` 
or 
``SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;``
Kindly fix this date issue.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @demix007 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (demix007) [reply queries.sql]: ## Hello @t-yanick 👋🏼 , 

Thank you for reviewing this PR, recommending changes and giving useful insights into how to make the changes. You are amazing 💯

**Review by elmar8287 — APPROVED** [review]
Hi, @demix007.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- You have implemented the suggestion from the previous reviewer 👍  
- You made a good communication with reviewer 👍 

### Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by demix007** [comment]
## Hello @elmar8287 😎 , 

Thank you for taking out time to review and approve this PR. You are awesome!!!💯

**Pull request merged**